examples/crypto/private_key_ed25519.py

```
@@ -1,18 +1,21 @@
 """
+
 Example file: Working with Ed25519 PrivateKey using the PrivateKey class
-*WARNING* Ed25519 seeds should not be printed or exposed in a real‑world scenario
+*WARNING* Ed25519 seeds should not be printed or exposed in a real‑world scenario.
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)